pkg/services/searchV2/index.go

```diff
@@ -50,6 +59,7 @@ type dashboardIndex struct {
 	eventStore   eventStore
 	logger       log.Logger
 	buildSignals chan int64
+	extenders    []DashboardIndexExtender
```

Comment:
given that this is internal, and we only really expect one implementation... should it just be
```suggestion
	extender    DashboardIndexExtender
```

Reply:
i think it depends on if it makes sense to eventually refactor any of the OSS code to use the extender interface

Reply:
i removed the slice in [125c56b](https://github.com/grafana/grafana/pull/49045/commits/125c56b93c00f530e2e49ed285ce7861d0731a2f)